projects/core/src/lib/map/map-info/map-info.component.ts

```diff
@@ -0,0 +1,39 @@
+import {ChangeDetectionStrategy, Component, Input} from '@angular/core';
+import {ProjectService} from '../../project/project.service';
+import {MapService} from '../map.service';
+import {combineLatestWith, Observable} from 'rxjs';
+import {map} from 'rxjs/operators';
+import {SpatialReferenceService} from '../../spatial-references/spatial-reference.service';
+
+/**
+ * The `geoengine-map-info` displays information about the visible map(s).
+ */
+@Component({
+    selector: 'geoengine-map-info',
+    templateUrl: 'map-info.component.html',
+    styleUrls: ['map-info.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class MapInfoComponent {
+    @Input()
+    public bottom!: number;
+
+    constructor(
+        private mapService: MapService,
+        private spatialReferenceService: SpatialReferenceService,
+        private projectService: ProjectService,
+    ) {}
+
+    info(): Observable<string> {
+        return this.projectService.getSpatialReferenceStream().pipe(
+            map((srs) => this.spatialReferenceService.getOlProjection(srs).getUnits()),
+            combineLatestWith(this.mapService.getViewportSizeStream()),
+            map(
+                ([unit, viewport]) =>
+                    `Resolution: ${viewport.resolution} ${unit}/px
+                     x: (${viewport.extent[0]}, ${viewport.extent[2]})
+                     y: (${viewport.extent[1]}, ${viewport.extent[3]})`,
```

Comment:
maybe it would make more sense to show min and max (of x and y) on top of each other and aligned?

like

min: x: -180 y: -90
max: x:  180 y:  90

then I think it's more easily visible how large the bbox is in each dimension. But this only makes sense if the numbers are aligned though.

Reply:
https://github.com/geo-engine/geoengine-ui/pull/720/commits/2ac20ad03521e0f6d9bbd9c750a252005a324b2e

Reply:
it's not enough to add a plus sign in front to properly align the numbers:

![image](https://github.com/geo-engine/geoengine-ui/assets/10283892/2284e49b-f6b7-499d-b7a1-493b4da6ab4e)

either we live with the numbers not being properly aligned, then we can remove the plus sign
or we properly align it. But then I guess we would need a monospace font and right-align the numbers, because then the decimal point would always be at the same place.

I'm fine with either solution.

Reply:
https://github.com/geo-engine/geoengine-ui/pull/720/commits/b5643386f00df93ec55b0986b052a1d7365aa027